docs/samples/browser-plugin-meetings/app.js

```diff
@@ -434,12 +434,18 @@ meetingsListElm.onclick = (e) => {
     document.getElementById('btn-join').disabled = true;
     document.getElementById('btn-join-media').disabled = true;
   }
-  else if (meeting && meeting.passwordStatus === 'UNKNOWN') {
+  else if (meeting && (meeting.passwordStatus === 'UNKNOWN' && meeting.destinationType!="")) {
     meetingsJoinPinElm.disabled = true;
     verifyPasswordElm.disabled = true;
     document.getElementById('btn-join').disabled = true;
     document.getElementById('btn-join-media').disabled = true;
   }
+  else if(meeting && (meeting.passwordStatus === 'UNKNOWN' && meeting.destinationType==="")) {
```

Comment:
Same here

Reply:
@Shreyas281299 ,That was I had keep an empty array because of the response of the meeting object.When I was given the Person ID or Email as destination type and click on create meeting button.The meeting object given the response as json and we stored it as the meeting variables.From that we taken Password Status and Destination type.For Person_ID or Email the destination type is "",so I had keep an empty array.I had tried to keep the Destination type as the option we selected but it won't Working to create a meeting and giving the meeting object where destination type is empty array.

Reply:
For Other SIP URI and Conversation URL the destination type is not null and password status was also different,so I had seen at different flow and keep it as an empty array